### DISCUSSION TITLE
Welcome to MineSweeper Discussions!

### DISCUSSION THREAD

**Discussion by juscghwe** [juscghwe/MineSweeper#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Stonedbybread** [comment]
This is cool I love minesweeper

**Reply by juscghwe** [reply]
Nice to hear. At the moment the project is still under development with slow progress bcs I'm just starting to learn C++ and try to teach myself industry standards with projects like this. Feel free to roam around. I'm open to criticism and feedback. And I'll make sure to let you know when I have a finished build to try out.
